akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerSpec.scala

```diff
@@ -1251,9 +1251,12 @@ class HttpServerSpec extends AkkaSpec(
           def expectDefaultEntityWithSizeError(limit: Int, actualSize: Int) =
             inside(request) {
               case HttpRequest(POST, _, _, entity @ HttpEntity.Default(_, `actualSize`, _), _) =>
-                val error = the[Exception]
-                  .thrownBy(entity.dataBytes.runFold(ByteString.empty)(_ ++ _).awaitResult(100.millis.dilated))
-                  .getCause
+                val origError = the[Exception]
+                  .thrownBy(entity.dataBytes.runFold(ByteString.empty)(_ ++ _).awaitResult(3.seconds.dilated))
+
+                log.error(origError, "Original Error")
+                val error = origError.getCause
```

Comment:
`getCause` returned zero here which probably means that `awaitResult` failed with a timeout after 100 millis which seems quite strict.